src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -439,6 +439,16 @@ public void testSetters() {
         }
     }
 
+    @Test
+    public void testSetSeverity() throws Exception {
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+        final DefaultConfiguration checkConfig = createModuleConfig(LineLengthCheck.class);
+        checkConfig.addProperty("max", "75");
+        final DefaultConfiguration checkerConfig = createRootConfig(checkConfig);
+        checkerConfig.addProperty("severity", "ignore");
+        verify(checkerConfig, getPath("InputCheckerTestSeverity.java"), expected);
```

Comment:
Please add empty line before this line.
It is better to separate configuration/preparation and execution

Reply:
I removed the empty lines in this test cuz the file was exceeding the limit of 2000 lines, it was giving error about it.